tuxedo-template-runtime/src/lib.rs

```diff
@@ -119,34 +119,34 @@ impl Default for GenesisConfig {
         const ANDREW_PUB_KEY_BYTES: [u8; 32] =
             hex!("baa81e58b1b4d053c2e86d93045765036f9d265c7dfe8b9693bbc2c0f048d93a");
 
-        // Initial Config just for a Money UTXO
-        GenesisConfig {
-            genesis_utxos: vec![
+        let genesis_transactions = vec![Transaction {
+            inputs: vec![],
+            peeks: vec![],
+            outputs: vec![
                 Output {
                     verifier: OuterVerifier::SigCheck(SigCheck {
                         owner_pubkey: SHAWN_PUB_KEY_BYTES.into(),
                     }),
-                    payload: DynamicallyTypedData {
-                        data: 100u128.encode(),
-                        type_id: <money::Coin<0> as UtxoData>::TYPE_ID,
-                    },
+                    payload: money::Coin::<0>(100).into(),
                 },
                 Output {
                     verifier: OuterVerifier::ThresholdMultiSignature(ThresholdMultiSignature {
                         threshold: 1,
                         signatories: vec![SHAWN_PUB_KEY_BYTES.into(), ANDREW_PUB_KEY_BYTES.into()],
                     }),
-                    payload: DynamicallyTypedData {
-                        data: 100u128.encode(),
-                        type_id: <money::Coin<0> as UtxoData>::TYPE_ID,
-                    },
+                    payload: money::Coin::<0>(100).into(),
                 },
             ],
+            checker: money::MoneyConstraintChecker::Mint.into(),
+        }];
+
+        GenesisConfig {
+            genesis_transactions,
         }
 
-        // TODO: Initial UTXO for Kitties
+        // TODO: Initial Transactions for Kitties
 
-        // TODO: Initial UTXO for Existence
+        // TODO: Initial Transactions for Existence
```

Comment:
Seems like a good time to resolve these. I think any example from any other piece would be sufficient. So maybe add a single PoE claim to genesis block, or a single amoeba creation to the genesis block.